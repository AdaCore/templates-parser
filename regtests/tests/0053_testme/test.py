from test_support import *

run('testme', ["testme49.tmplt"])
