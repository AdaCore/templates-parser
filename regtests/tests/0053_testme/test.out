
raised TEMPLATES_PARSER.TEMPLATE_ERROR : Can't have a positional parameter after a named one, in testme49.tmplt at line 1
